### PR TITLE
Added recipe for hachoir-metadata

### PR DESCRIPTION
**Pull request by pmlandwehr**
This is the final `hachoir-*` module required for the build of `girder` (#1039).

### CODE REVIEW

**Comment by pmlandwehr** [comment]
Close and reopen - windows builds hadn't yet been pushed.

File: recipes/hachoir-metadata/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - hachoir-parser
+
+test:
+  imports:
+    - hachoir_metadata
+    - hachoir_metadata.qt
+
+about:
+  home: http://bitbucket.org/haypo/hachoir/wiki/hachoir-metadata
+  license: GNU General Public License (GPL)
```
Comment (jakirkham) [inline recipes/hachoir-metadata/meta.yaml]: Please specify version (e.g. GPL 2).

**Comment by jakirkham** [comment]
LGTM

**Comment by pmlandwehr** [comment]
(ping @ocefpaf)

**Comment by jakirkham** [comment]
He's in transit. Sorry was waiting to see if anyone else wanted to say anything. Happy to merge.

**Comment by jakirkham** [comment]
@haypo, just letting you know we have added `hachoir-metadata` to [conda-forge]( http://conda-forge.github.io/ ). This will allow users to easily install this with [`conda`]( http://conda.pydata.org/docs/ ) (`conda install -c conda-forge hachoir-metadata`). The maintenance burden here is light as all testing releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe in [this repo]( https://github.com/conda-forge/hachoir-metadata-feedstock ). If you have any interest in being added as a maintainer (to make new releases for instance), we would love to have you on board. Thanks for your work on `hachoir-metadata`. 😄

**Pull request merged**